### PR TITLE
add some package to code-generator

### PR DESCRIPTION
**Pull request by fisherxu**
**What this PR does / why we need it**:
add some package to code-generator and regenerated files.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
/cc @sttts 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go [hunk 1]
```diff
@@ -74,6 +74,7 @@ func New() *Generator {
 			`+k8s.io/apimachinery/pkg/runtime`,
 			`k8s.io/apimachinery/pkg/apis/meta/v1`,
 			`k8s.io/apimachinery/pkg/apis/meta/v1beta1`,
+			`k8s.io/apimachinery/pkg/apis/testapigroup/v1`,
```
Comment (sttts) [inline staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go]: this need a follow-up to remove these hardcoded package from k8s.io/code-generator

File: staging/src/k8s.io/code-generator/_examples/apiserver/apis/example/v1/zz_generated.conversions.go [hunk 2]
```diff
@@ -1,113 +0,0 @@
-// +build !ignore_autogenerated
-
-/*
-Copyright 2017 The Kubernetes Authors.
```
Comment (sttts) [inline staging/src/k8s.io/code-generator/_examples/apiserver/apis/example/v1/zz_generated.conversions.go]: why is this gone?

File: staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/zz_generated.conversion.go [hunk 3]
```diff
@@ -95,6 +96,8 @@ func autoConvert_internalversion_ListOptions_To_v1_ListOptions(in *ListOptions,
 	out.Watch = in.Watch
 	out.ResourceVersion = in.ResourceVersion
 	out.TimeoutSeconds = (*int64)(unsafe.Pointer(in.TimeoutSeconds))
+	out.Limit = in.Limit
```
[the remark this answers is not in the document]
Comment (fisherxu) [reply staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/zz_generated.conversion.go]: yeah, maybe long time not regenerated.

File: staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go (hunk 1 above)
Comment (fisherxu) [reply staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go]: can add a tag for protobuf .

File: staging/src/k8s.io/code-generator/_examples/apiserver/apis/example/v1/zz_generated.conversions.go (hunk 2 above)
Comment (fisherxu) [reply staging/src/k8s.io/code-generator/_examples/apiserver/apis/example/v1/zz_generated.conversions.go]: filename is conversion(s), also has same file zz_generated.conversion.go in the dir, don't know why..

Comment (sttts) [reply staging/src/k8s.io/code-generator/_examples/apiserver/apis/example/v1/zz_generated.conversions.go]: this one is concerning, shouldn't happen

Comment (sttts) [reply staging/src/k8s.io/code-generator/_examples/apiserver/apis/example/v1/zz_generated.conversions.go]: oh, did the name change? If we still have zz_generated.conversion.go here, it's probably ok.

Comment (fisherxu) [reply staging/src/k8s.io/code-generator/_examples/apiserver/apis/example/v1/zz_generated.conversions.go]: Can have a look in the dir https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/code-generator/_examples/apiserver/apis/example/v1   :)

**Comment by fisherxu** [comment]
Kindly ping @sttts for approval this :)

File: staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go (hunk 1 above)
Comment (sttts) [reply staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go]: Yes, like for deepcopy.

Comment (fisherxu) [reply staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go]: Ok, I can try send a follow-up PR to remove the hardcoded package from k8s.io/code-generator

Comment (sttts) [reply staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/cmd.go]: :+1:

**Pull request merged**